### PR TITLE
Design welcome page

### PR DESCRIPTION
**Pull request by ilarne**

### CODE REVIEW

**Review by TimRobinson1 — APPROVED** [review]
Seems pretty straightforward.  Indentation looks good, far as I can see.  Let's get this cohort image live!

**Comment by BertZZ** [comment]
I know literally no CSS. So this looks great to me

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -25,3 +25,15 @@ ready(function() {
...
+var slides = document.querySelectorAll('#slides .slide');
+var currentSlide = 0;
+var slideInterval = setInterval(nextSlide,3500);
+
+function nextSlide() {
+    slides[currentSlide].className = 'slide';
+    currentSlide = (currentSlide+1)%slides.length;
+    slides[currentSlide].className = 'slide showing';
+};
+});
```
Comment (neoeno) [inline app/assets/javascripts/application.js]: Indentation

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -25,3 +25,15 @@ ready(function() {
...
 });
+
+ready(function() {
+var slides = document.querySelectorAll('#slides .slide');
+var currentSlide = 0;
+var slideInterval = setInterval(nextSlide,3500);
+
+function nextSlide() {
+    slides[currentSlide].className = 'slide';
+    currentSlide = (currentSlide+1)%slides.length;
```
Comment (neoeno) [inline app/assets/javascripts/application.js]: It'd be better not to push all the operators together like this, prefer:

```javascript
currentSlide = (currentSlide + 1) % slides.length;
```

File: app/assets/stylesheets/footer.css [hunk 3]
```diff
@@ -1,33 +1,22 @@
...
-   color: #fff;
- }
+.footer p {
+  color: #fff;
+  display: block;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.footer_link {
```
Comment (neoeno) [inline app/assets/stylesheets/footer.css]: `.footer-link`

File: app/assets/stylesheets/forms.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
   width: 280px;
   padding: 33px 25px 29px;
   background: #964040;
-  border-bottom: 1px solid #c4c4c4;
+  // border-bottom: 1px solid #c4c4c4;
```
Comment (neoeno) [inline app/assets/stylesheets/forms.scss]: Delete

File: app/assets/stylesheets/user_new.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
+body {
+  background-image: asset-data-url("MAbg5.png");
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.new-user {
+  margin: 0 auto;
+  margin-top: 20vh;
+
```
Comment (neoeno) [inline app/assets/stylesheets/user_new.scss]: Empty space

File: app/assets/stylesheets/user_new.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  font-size: 30px;
+}
+
+.float-left {
+  float: left;
+  width: 50%;
+  margin-top: 20vh;
+}
+
+#slides {
```
Comment (neoeno) [inline app/assets/stylesheets/user_new.scss]: Prefer classes to IDs when it comes to styling

File: app/assets/stylesheets/user_new.scss [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+.slide {
+  position: absolute;
+  left: 0px;
+  top: 0px;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  z-index: 1;
+  transition: opacity 2s;
```
Comment (neoeno) [inline app/assets/stylesheets/user_new.scss:50]: Nice!

File: app/views/users/new.html.erb [hunk 8]
```diff
@@ -1,6 +1,20 @@
...
+    <ul id="slides">
+      <li class="slide showing">"It's the greatest social network. It has the best posts.
+        The greatest photos. It's gonna be yee-uuuuge."</li>
+      <li class="slide">"My favourite social network, it is!"</li>
+      <li class="slide">"I met my dog walker on Acebook!"</li>
+      <li class="slide">"Hmm, interesting."</li>
+      <li class="slide">"Finally, another way to feel socially inadequate!"</li>
+    </ul>
+  </div>
+</div>
```
Comment (neoeno) [inline app/views/users/new.html.erb:15]: Love it!

**Review by neoeno — CHANGES_REQUESTED** [review]
Love it — just a few code style and formatting comments for good measure!

**Comment by ilarne** [comment]
Changes made!

**Review by neoeno — APPROVED** [review]

**Comment by neoeno** [comment]
:+1: :+1: :+1: 🚢

**Pull request merged**